src/nvim/testdir/script_util.vim

```diff
@@ -0,0 +1,69 @@
+" Functions shared by the tests for Vim Script
```

Comment:
Add in the commit message that this came from https://github.com/vim/vim/commit/a6296200bd5191bab7efcdcc16c9e79eb498e8e0.

Reply:
Do you want this whole patch (8.2.1366: test 49 is old style) ported as another commit?

Reply:
Porting that patch right now will complicate this PR because of older unmerged patches.

Mention where this file came from and why it was included here.

Reply:
Thanks!  I've updated the PR.